windows/rngallery/MainPage.cpp

```diff
@@ -18,5 +19,17 @@ namespace winrt::rngallery::implementation
         ReactRootView().ReactNativeHost(app->Host());
         // Workaround for https://github.com/microsoft/react-native-windows/issues/6287
         ReactRootView().IsPerspectiveEnabled(false);
+        auto version = Windows::ApplicationModel::Package::Current().Id().Version();
+        ReactRootView().InitialProps([&](const IJSValueWriter& writer) {
+            writer.WriteObjectBegin();
+            writer.WritePropertyName(L"MajorVersion");
+            writer.WriteInt64(int(version.Major));
```

Comment:
```suggestion
            writer.WriteInt64(static_cast<int>(version.Major));
```
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c146-use-dynamic_cast-where-class-hierarchy-navigation-is-unavoidable
Type [appears to be](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.packageversion.major?view=winrt-20348) a ushort/UInt16, so it should cast to a 64 bit int just fine. But for all of these you should use the named cast.